pkgs/applications/file-managers/mc/default.nix

```diff
@@ -100,7 +100,7 @@ stdenv.mkDerivation rec {
  '';
 
   meta = with lib; {
-    description = "File Manager and User Shell for the GNU Project";
+    description = "File Manager and User Shell for the GNU Project - midnight-commander";
```

Comment:
Definitely good idea, but I'd prefer the description to be more human readable:
```suggestion
    description = "File Manager and User Shell for the GNU Project, known as Midnight Commander";
```

Reply:
Could you include this @06kellyjac ? It's ready to merge otherwise.

Reply:
Up 😇 that would be a welcome improvement